server/src/config.js

```diff
@@ -0,0 +1,31 @@
+require("envc")();
```

Comment:
What does envc do?  A comment might be helpful, as the construct is pretty mysterious looking ;)